src/ws/cockpithandlers.c

```diff
@@ -627,7 +627,7 @@ cockpit_handler_default (CockpitWebServer *server,
   /* Stuff in /cockpit or /cockpit+xxx */
   if (resource)
     {
-      cockpit_web_response_skip_path (response);
+      g_assert (cockpit_web_response_skip_path (response));
```

Comment:
Is it correct to use g_assert here?

https://developer.gnome.org/glib/stable/glib-Testing.html#g-assert says:
> The macro can be turned off in final releases of code by defining 
> G_DISABLE_ASSERT when compiling the application, so code must not 
> depend on any side effects from expr .

Reply:
We don't depend on it really. The code 7 lines above already ensures that response's path starts with `/cockpit, so at this point this really should not fail. The assert will catch this during tests, and it's fine to remove it. This is really mostly a pacifier for coverity.